tests/rustdoc-gui/src/macro_expansion/lib.rs

```diff
@@ -0,0 +1,51 @@
+// Test crate used to check the `--generate-macro-expansion` option.
+//@ compile-flags: -Zunstable-options --generate-macro-expansion --generate-link-to-definition
+
```

Comment:
Please make this test a multi-file crate, so we can ensure those are handled properly.

Reply:
Sure!